packages/hash/api/src/graph/system-types.ts

```diff
@@ -396,17 +383,6 @@ const textEntityTypeInitializer = async (graphApi: GraphApi) => {
   })(graphApi);
 };
 
-/**
- * @todo: remove this dummy entity type once we are able to define the block data link type without it
- * @see https://app.asana.com/0/1202805690238892/1203015527055368/f
- */
-const dummyEntityTypeInitializer = async (graphApi: GraphApi) => {
-  return entityTypeInitializer({
-    ...types.entityType.dummy,
```

Comment:
I think it should also be removed from [the `ontology-types.ts` file](https://github.com/hashintel/hash/blob/93a13184191c24979f7fc941deee754da7f796cb/packages/hash/shared/src/ontology-types.ts#L113-L116)

Reply:
Thanks, kind of annoyed I missed these 😅